includes/Media.php

```diff
@@ -74,6 +74,8 @@ class Media {
 
 	/**
 	 * Init.
+	 *
+	 * @todo refactor
```

Comment:
?

Reply:
Not happy with the way the PHP code is structured at the moment 🙂

Reply:
Well that is clear, why is it being added in this unrelated PR.

Reply:
Oh, I just realized I added this to `Media.php` instead of `Story_Post_Type.php` (which already has that comment apparently). I can definitely omit this here 👍